### PR TITLE
Search Over All Aggregate Fields 

### PR DESCRIPTION
**Pull request by blewis11**
The current search functionality in Periskop only searches over the error aggregate keys, this PR means we can now search over the entire error (url, error message, etc) 🔍

### CODE REVIEW

File: web/src/components/Error.tsx [hunk 1]
```diff
@@ -247,10 +246,17 @@ const ErrorComponent: React.FC<Props> = (props) => {
 }
 
 const mapStateToProps = (state: StoreState) => {
+  const searchTerm = state.errorsReducer.searchTerm
+  const activeError = state.errorsReducer.activeError
+
+  // clone the aggregate so that we don't mutate the state
+  const errorCopy = JSON.parse(JSON.stringify(activeError))
```
Comment (jcreixell) [inline web/src/components/Error.tsx]: I don't think we need any of this copying (same on `errors.ts`)

**Review by jcreixell — APPROVED** [review]
See my comment, otherwise LGTM. Let's see how well it performs for large numbers of errors 🔝

**Comment by marctc** [comment]
> Let's see how well it performs for large numbers of errors top

I tested locally using config of a project with a larg number of errors. Didn't notice any performance issues

**Review by marctc — APPROVED** [review]
Great! Maybe we can change the input message of the search bar

**Review by jcreixell — APPROVED** [review]
amazing, great work

**Pull request merged**